### PR TITLE
Rebased, and merged all the commits into one...

### PR DESCRIPTION
**Pull request by gourabnagDev**
Added the Google plugin.

### CODE REVIEW

**Comment by gourabnagDev** [comment]
":p added the wotd plugin back to the list in wotd plugin."*
":p added the wotd plugin back to the list"

**Comment by gourabnagDev** [comment]
@kelunik @PeeHaa ^

File: config/config.sample.yml [hunk 1]
```diff
@@ -20,3 +20,5 @@ twitter:
   consumerSecret: YourConsumerSecret
   accessToken: YourAccessToken
   accessTokenSecret: YourTokenSecret
+bilty:
```
Comment (staabm) [inline config/config.sample.yml]: missing indent

**Comment by gourabnagDev** [comment]
\o/

**Pull request merged**